compiler/src/dmd/tokens.d

```diff
@@ -582,6 +583,7 @@ private immutable TOK[] keywords =
     TOK._import,
     TOK.__cdecl,
     TOK.__declspec,
+    TOK._stdcall,
```

Comment:
Could this instead be handled by `importc.h` as a macro to sanitize the vagary of names for the same keyword?

Reply:
Hmm. I was looking for another option though. Let me try it.